rest/models/snapshot.go

```diff
@@ -224,6 +224,7 @@ type TickerSnapshot struct {
 	TodaysChange     float64           `json:"todaysChange,omitempty"`
 	TodaysChangePerc float64           `json:"todaysChangePerc,omitempty"`
 	Updated          Nanos             `json:"updated,omitempty"`
+	FairMarketValue  float64           `json:"fmv,omitempty"`
```

Comment:
done

Reply:
oh wait the coverage still hasn't changed, hmmm

Reply:
would unmarshaling an additional field change test coverage?

Reply:
i don't fully understand how to trigger the test coverage runner again. it seems to run right after opening a PR but then doesn't after additional commits?

Reply:
looks like it did run again: https://github.com/polygon-io/client-go/actions/runs/6564761758

Maybe just didn't update the readme because the value is the same

Reply:
ahh okay, makes sense. the coverage only went down a bit because of the websocket additions. lgtm then